packages/mobile/src/store/offline-downloads/sagas/migrateOfflineDataPathSaga.ts

```diff
@@ -0,0 +1,95 @@
+import path from 'path'
+
+import { difference } from 'lodash'
+import RNFS from 'react-native-fs'
+import RNFetchBlob from 'rn-fetch-blob'
+import { call, put, select } from 'typed-redux-saga'
+
+import { downloadsRoot } from 'app/services/offline-downloader'
+import {
+  getOfflineCollectionsStatus,
+  getOfflineQueue,
+  getOfflineTrackStatus
+} from 'app/store/offline-downloads/selectors'
+import type { OfflineJob } from 'app/store/offline-downloads/slice'
+import { redownloadOfflineItems } from 'app/store/offline-downloads/slice'
+
+import { DOWNLOAD_REASON_FAVORITES } from '../constants'
+
+import { getIsOfflineEnabled } from './getIsOfflineEnabled'
+const {
+  fs: { dirs, unlink, exists }
+} = RNFetchBlob
+
+// Current migration: 4/3/2023
+// dirs.DocumentDir -> dirs.CacheDir
+const legacyDownloadsRoot = path.join(dirs.DocumentDir, 'downloads')
+
+// Move downloads from legacy storage location to the updated path (currently: /Documents -> /Caches)
+export function* migrateOfflineDataPathSaga() {
+  const isOfflineModeEnabled = yield* call(getIsOfflineEnabled)
+  if (!isOfflineModeEnabled) return
+
+  const legacyFilesExist = yield* call(exists, legacyDownloadsRoot)
+  if (!legacyFilesExist) return // TODO: fire action?
```

Comment:
just wondering if this should be resolved?

Reply:
oh yeah I think it's unnecessary. ty!